app/models/activity.rb

```diff
@@ -120,8 +123,8 @@ def has_implementing_organisations?
   end
 
   def parent_activities
-    return [activity] if programme?
-    return [activity.activity, activity] if project?
+    return [parent_activity] if programme?
```

Comment:
It's annoying we don't have control over these names. Can we maybe wrap the model mixin in one of our own that aliases `activity` to `event` instead, and disables public access to `activity`? I think the rename to `parent_activity` is a good one regardless, because it describes the relationship more clearly.

Reply:
I'd like to do it but I'm not entirely sure how either 😬